ecommerce/templates/edx/checkout/receipt.html

```diff
@@ -106,6 +106,11 @@ <h2>{% trans "Order Information" as tmsg %}{{ tmsg | force_escape }}</h2>
                 <dd class="line-price price">{{ line.unit_price_incl_tax|currency:order.currency }}</dd>
               </div>
             {% endfor %}
+             <form style="display: none;"  name="aw_basket_form">
+               <textarea wrap="physical" id="aw_basket">
+                 {{ product_tracking }}
+               </textarea>
```

Comment:
Make this one line otherwise, it will add spaces at the start of the value.
```suggestion
               <textarea wrap="physical" id="aw_basket">{{ product_tracking }}</textarea>
```